examples/factorial.qk

```diff
@@ -1,20 +1,11 @@
-(** vim: syntax=Scala *)
+import from console
```

Comment:
is this already working?

Reply:
No yet. This depends up on #115

Reply:
As I said in #115, the `from` should come first to help with IDE completion.

Reply:
@joaolucasl yeah! I forgot it here! I'm gonna fix!

Reply:
🎉